src/voided-disputes.js

```diff
@@ -0,0 +1,34 @@
+import { networks, RINKEBY_COURT, RINKEBY_USABILITY_COURT } from './networks'
+import { getNetworkName } from './lib/web3-utils'
+import env from './environment'
+
+const DEFAULT_VOID_TEXT = ''
+const DEFAULT_VOID_LINK = ''
+
+export const VoidedDisputes = {
+  rpc: new Map([[networks.rpc.court, []]]),
+  ropsten: new Map([[networks.ropsten.court, []]]),
+  rinkeby: new Map([
+    [RINKEBY_COURT, []],
+    [RINKEBY_USABILITY_COURT, []],
+  ]),
+  mainnet: new Map([
+    [
+      networks.mainnet.court,
+      [
```

Comment:
What do you think about making this a map of `dispute id -> voided information`?